mass_mailing_event/models/event_state.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields
+
+
+class EventState(models.Model):
+    _name = 'event.state'
+
+    name = fields.Char(required=True)
```

Comment:
we actually can do this: Create an `AbstractModel`, make `search` there return the values of the selection, and `read` return a dict/list of dicts crafted from the selection's display values. Last time I tried this, nobody cared about the type of the ids, if this changed, you might have to translate the values to ints first and retranslate on read. This should be a nice exercise in bending the ORM a bit to our will.
Then as there are no database records, you need to save the values in a serialized fields, and wrap a function field around this that reads/writes to this serialized field.

This way, you're resilient against states added/removed from the registration model.

The model stands for event registration states, so I think `event.registration.state` whould be a better name